.circleci/config.yml

```diff
@@ -79,7 +79,7 @@ jobs:
     <<: *defaults
     <<: *version
     docker:
-      - image: hyperledger/iroha-docker-develop:v1
+      - image: hyperledger/iroha-docker-develop:v2
```

Comment:
Maybe it is better to add a global variable with the image name?

Reply:
It will not be  changed in this PR.